sirepo/template/elegant.py

```diff
@@ -5,6 +5,7 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from __future__ import absolute_import, division, print_function
+import ast
```

Comment:
I think this import can be removed as well as the ast related imports in template/opal